src/test/scala/uk/gov/nationalarchives/aws/utils/StepFunctionUtilsTest.scala

```diff
@@ -69,4 +68,29 @@ class StepFunctionUtilsTest extends AnyFlatSpec with MockitoSugar with EitherVal
     val response = stepFunctionUtils.sendTaskFailureRequest(taskToken, causeMessage).attempt.unsafeRunSync()
     response.left.value.getMessage should equal("Task failure request failed")
   }
+
+  "the sendTaskHeartbeat method" should "be called with the correct parameters" in {
+    val stepFunctionClient = Mockito.mock(classOf[SfnAsyncClient])
+    val stepFunctionUtils = StepFunctionUtils(stepFunctionClient)
+
+    val argumentCaptor: ArgumentCaptor[SendTaskHeartbeatRequest] = ArgumentCaptor.forClass(classOf[SendTaskHeartbeatRequest])
+    val response = SendTaskHeartbeatResponse.builder.build
+
+    when(stepFunctionClient.sendTaskHeartbeat(argumentCaptor.capture())).thenReturn(CompletableFuture.completedFuture(response))
+
+    stepFunctionUtils.sendTaskHeartbeat(taskToken).unsafeRunSync()
+
+    val request: SendTaskHeartbeatRequest = argumentCaptor.getValue
+    request.taskToken should equal("taskToken")
+  }
+
+  "The sendTaskHeartbeat method" should "return an error if the request fails" in {
```

Comment:
Change to lower case "t"

Reply:
I've changed this one. There was a couple of other upper case ones in this file which I've changed as well. This is for that ticket TDR-1318. There's a PR coming after this is merged to change the consignment export and the step function.